hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBBatchOperation.java

```diff
@@ -140,15 +143,17 @@ private class FamilyCache {
        * and the dbValue type is {@link Object}.
```

Comment:
Could you update its javadoc correspondingly?

e.g. (incomplete)

```suggestion
       * and the dbValue type is a {@link Pair} of {@link Op} and {@link Object}.
```